static/js/containers/ProfilePage_test.js

```diff
@@ -256,7 +245,8 @@ describe("ProfilePage", function() {
     [EMPLOYMENT_STEP, 'EmploymentTab'],
   ]) {
     it(`sends the right props to tab components for step ${step}`, () => {
-      return renderComponent(`/profile/${step}`, SUCCESS_ACTIONS).then(([wrapper]) => {
+      let actions = step === PERSONAL_STEP ? SUCCESS_ACTIONS.concat(SET_PROGRAM) : SUCCESS_ACTIONS;
```

Comment:
Yeah, I agree that would make a *lot* of these tests easier. We wrote the redux-asserts library, so we could definitely add that functionality if we wanted!